client/my-sites/checkout/src/lib/get-flow-plan-features.ts

```diff
@@ -2,13 +2,118 @@ import {
 	applyTestFiltersToPlansList,
 	getPlan,
 	FEATURE_CUSTOM_DOMAIN,
+	IncompleteWPcomPlan,
 } from '@automattic/calypso-products';
+import {
+	NEWSLETTER_FLOW,
+	isLinkInBioFlow,
+	isAnyHostingFlow,
+	isNewsletterOrLinkInBioFlow,
+	isBlogOnboardingFlow,
+} from '@automattic/onboarding';
 import { ResponseCartProduct } from '@automattic/shopping-cart';
 import { getPlanFeaturesObject } from 'calypso/lib/plans/features-list';
-import {
-	getHighlightedFeatures,
-	getPlanFeatureAccessor,
-} from 'calypso/my-sites/plan-features/util';
+
```

Comment:
I understand that this code was moved from `client/my-sites/plan-features/util.ts`, and the decision to place this inline was taken probably because these functions are only used in this file.

However, this code has signup/onboarding-specific logic that seems out of place in a checkout library function. I don't think checkout should know about all the specific flows and how to build the accessor functions of each flow. Do you think we can move these parts to somewhere in `@automattic/onboarding` as independent & testable functions?

Reply:
Actually this is not onboarding specific logic. This is checkout specific logic. The features `highlighted` on the **checkout page** can depend on which `flow` they come from. Shouldn't this code live closer to checkout? This feature highlighting happens only at the checkout page.

Reply:
Hmm, that's true. For now, it's better for this code to live here, and we can always separate out this if required elsewhere.